packages/did-ethr-resolver/src/implementations/operator.ts

```diff
@@ -92,7 +93,7 @@ export class Operator extends Resolver implements IOperator {
       algo: Algorithms.Secp256k1,
       type: PubKeyType.VerificationKey2018,
       encoding: Encoding.HEX,
-      value: { publicKey: `0x${this.getPublicKey()}`, tag: KeyTags.OWNER },
+      value: { id: uuid(), publicKey: `0x${this.getPublicKey()}`, tag: KeyTags.OWNER },
```

Comment:
@JGiter Do you think it makes sense to use `uuid` as the default id for publicKeys?

Reply:
actually id of public constructed from `did` and public key tag: https://github.com/energywebfoundation/ew-did-registry/blob/b3e025141fb085b1a468a94de775178014fa8c86/packages/did-ethr-resolver/src/functions/functions.ts#L109 Why not use same approach for service endpoint id? For example like `${did}#${serviceEndpoint}`